src/index.ts

```diff
@@ -20,7 +20,7 @@ import { runTests } from './testing';
 import { postProcessCairoFile } from './utils/postCairoWrite';
 
 export type CompilationOptions = {
-  warnings: boolean;
+  warnings?: boolean;
```

Comment:
Why do we make this property nullable? Isn't the default behavior just to take it as false? If that's the case we avoid "options.warnings || false" further in the code